GOVERNANCE.md

```diff
@@ -0,0 +1,177 @@
+
+👉 **All community members must follow the [Code of Conduct (CoC)](https://github.com/gatsby-uc/plugins/blob/main/CODE_OF_CONDUCT.md).**
+Consequences for CoC violations are detailed in [Moderation](#moderation).
+
+> Note: This community is very new. This doc will intentionally be vague at times and is for the moment only intended as rough idea of how things will work. This is intended so that as the community grows we together can define how we wish to govern ourselves.
+
+## Get Involved
+
+**Anything that supports the GUC community is a valuable contribution.** All types of contribution are meaningful, from code to documentation to blog posts. Anyone can become an Contributor (yes, even you!). Our goal is to recognize all contributors to the GUC regardless of skill, experience, or background.
+
+## Contributor Levels
+
+We recognize different degrees of contribution as levels, and most levels can be reached regardless of coding skill or years of experience. The two most important things that we look for in contributors are:
+
+- **Being here** - Everyone's time is valuable, and the fact that you're here and contributing to GUC plugins is amazing! Thank you for being a part of this journey with us.
+
+- **Being a positive member of our community** - Go above and beyond our Code of Conduct, and commit to healthy communication in pull requests, issue discussions, Discord conversations, and interactions outside of our community (eg: no Twitter bullies)
+
+Each level unlocks new privileges and responsibilities on Discord and GitHub. Below is a summary of each contributor level:
+
+### Level 1 - Contributor
+
+Have you done something (big or small) to contribute to the health, success, or growth of the GUC or its plugins? Congratulations, you're officially recognized as a contributor to the project!
+
+#### Examples of recognized contributions
+
+- **GitHub:** Submitting a merged pull request
+- **GitHub:** Filing a detailed bug report or RFC
+- **GitHub:** Updating documentation!
+- Helping people on GitHub, Discord, etc.
+- Answering questions on Stack Overflow, Twitter, etc.
+- Blogging, Vlogging, Podcasting, and Livestreaming about GUC projects
+- This list is incomplete! Similar contributions are also recognized.
+
+#### Privileges
+
+At this time we have no specifics for this role. as the community grows we anticipate privleges to come. If you have ideas here please let us know!
+
+#### Responsibilities
+
+This role does not require any extra responsibilities or time commitment. We hope you stick around and keep participating!
+
+If you're interested in reaching the next level and becoming a Maintainer, you can begin to explore some of those responsibilities in the next section.
+
+#### Nomination Process
+
+n/a
+
+### Level 2 (L2) - Maintainer
+
+The **Maintainer** role is available to any contributor who wants to join the team and take part in the long-term maintenance of a plugin(s) in the GUC.
```

Comment:
```suggestion
The **Maintainer** role is available to any contributor who wants to join the team and take part in the long-term maintenance of plugins in the GUC.
```

From my standpoint it's also clear this way that it doesn't necessarily mean you maintain all plugins